packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -81,12 +77,6 @@ const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [is_switching]);
 
-    React.useEffect(() => {
-        if (typeof setIsOverlayShown === 'function') {
```

Comment:
Why do you getting rid of this?

Reply:
From what I understood, `is_overlay_shown ` is used to conditionally render `<ApiTokenOverlay />`. But since, we never actually `<ApiTokenOverlay />` component I removed `is_overlay_shown `and its related functions/variables wherever used in the file.